.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,40 @@
+# Contributing to VSLR UI Kit
+
+Thanks for being willing to contribute. Together we can all make our shared product better. Yay.
+
+## Did you find a bug?
+
+* **Ensure that the bug has not already been reported.** We try to track of issues on [Github](https://github.com/vivintsolar-oss/react-native-components/issues).
+* If you're unable to find an open issue addressing the problem, open a new one on [Github](https://github.com/vivintsolar-oss/react-native-components/issues/new). Be sure to include a title and clear description, as much relevant information as possible, and a **code sample** or an **executable test case**.
+
+## Did you write code for a feature, bug, etc.
+
+* Open a new Gitub pull request with the patch.
+* The PR template will outline all of the questions that a contributor should ask themself before asking a codeowner to review their PR for approval
+* Check off each item in the PR template checklist whether it applies or not as a sign that you've considered the question and complied appropriately for your contribution
+* If you want preliminary review of your ideas, feel free to slack a code owner, too.
+* When you're ready for a formal review, slack a code owner
+
+## Do you have questions
+
+* Ask them. Slack a code owner
```

Comment:
Who is them?

Reply:
Forgot to add the code owners, I'll add the link